README.md

```diff
@@ -107,15 +133,12 @@ http://groups.google.com/group/rgeo-users
 ### Acknowledgments
 
 [Daniel Azuma](http://www.daniel-azuma.com) created RGeo.
-[Tee Parham](http://twitter.com/teeparham) is the current maintainer.
-
-Development is supported by:
-
-* [Pirq](http://pirq.com)
-* [Neighborland](https://neighborland.com)
+[Tee Parham](http://twitter.com/teeparham) is a former maintainer.
+[Keith Doggett](http://www.github.com/keithdoggett) is a current maintainer.
+[Ulysse Buonomo](http://www.github.com/BuonOmo) is a current maintainer.
```

Comment:
Shouldn't we keep the development supported by and replace it by both our companies since in my case, most of my work time on this project is given by Klaxit. The li for mine would be:

```markdown
- [Klaxit](https://www.klaxit.com)
```

Reply:
Done. I'll add my company's website after we're done fixing it.